modules/distr-tracing-rn-new-features.adoc

```diff
@@ -13,10 +13,27 @@ Result – If changed, describe the current user experience.
 
 This release adds improvements related to the following components and concepts.
 
+== New features and enhancements {DTProductName} 2.3.1
+
+This release of {DTProductName} addresses Common Vulnerabilities and Exposures (CVEs) and bug fixes.
+
+=== Component versions supported in {DTProductName} version 2.3.1
+
+|===
+|Component |Version
+|Jaeger
+|1.30.2
```

Comment:
Is this the jaeger operator version right? not the jaeger component

Reply:
Yes the versions here are just the bundle versions. The upstream is still 1.30.0 and 0.44.0 for OTEL.

Reply:
Versions confirmed in the GUI
![OperatorVersions](https://user-images.githubusercontent.com/4106682/163417890-404de75e-6d0e-478b-928b-c4c8f42e8c15.png)